crates/turborepo-lib/src/config/mod.rs

```diff
@@ -60,6 +60,19 @@ pub struct UnnecessaryPackageTaskSyntaxError {
     pub text: NamedSource<String>,
 }
 
+#[derive(Debug, Error, Diagnostic)]
+#[diagnostic(
+    code(root_syntax_in_global_deps),
+    url("{}/messages/{}", TURBO_SITE, self.code().unwrap().to_string().to_case(Case::Kebab))
+)]
+#[error("$$ROOT$$ syntax is not allowed in globalDependencies")]
```

Comment:
```suggestion
#[error("$$ROOT$$ syntax is not allowed in globalDependencies, since globalDependencies is already relative to the root of the Workspace.")]
```